.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -53,3 +52,6 @@ To add a screenshot, please use one of the following formats and/or methods desc
 
 * https://help.github.com/en/articles/file-attachments-on-issues-and-pull-requests
 *
+
+**Will you be able to contribute a Pull Request for this issue?**  
```

Comment:
```suggestion
**Will you be able to contribute a pull request for this issue?**  
```

Reply:
changed with the last comment's rephrasing